recirq/quantum_chess/move.py

```diff
@@ -46,35 +46,38 @@ class Move:
     have a move type and variant that determines what kind of move this is
     (capture, exclusion, etc).
     """
-
     def __init__(self,
                  source: str,
                  target: str,
                  *,
                  source2: str = None,
                  target2: str = None,
                  move_type: enums.MoveType = None,
-                 move_variant: enums.MoveVariant = None):
+                 move_variant: enums.MoveVariant = None,
+                 measurement: bool = None):
```

Comment:
Should we use an int for this instead? Is any one measurement "truer" than the other? And we also won't have to deal with the conversion in post_select later on.

Reply:
That's a good thought. Int would also allow for the possibility of  multi outcome measurements on the future